### DISCUSSION TITLE
SDXL Safetensors not found -- Where Should They Be Put?

### DISCUSSION THREAD

**Discussion by AncientEpic** [vladmandic/sdnext#1767] [Q&A]
Getting this error:
`Loading model stabilityai/stable-diffusion-xl-base-0.9 text_encoder_2\model.fp16.safetensors not found`

Those files are located in the SD subfolder under models. Is that not where they're supposed to be?

**Comment by vladmandic** [comment]
if they are in standard safetensors format, in standard models/stable-diffusion
if they are in hf hub folder-type format, in models/diffusers
